src/cpp/benders/benders_core/BendersBase.cpp

```diff
@@ -520,4 +522,52 @@ std::string BendersBase::status_from_criterion() const
 	default:
 		return Output::STATUS_ERROR_C;
 	}
+}
+
+/*!
+ *  \brief Get path to slave problem mps file from options
+ */
+std::string BendersBase::get_slave_path(std::string const &slave_name) const
+{
+	return (Path(_options.INPUTROOT) / (slave_name + ".mps")).get_str();
+}
+
```

Comment:
Why not return a `Path` ? This remarks also holds for `BendersBase::get_structure_path` and `BendersBase::get_structure_path`.

Also, ".mps" should be a named variable/member function.